wiki/Do_you_really_want_to_ask_peppy/fr.md

```diff
@@ -0,0 +1,13 @@
+Salut, je comprends votre urgence de me contacter personnellement, mais je vous devez comprendre que ce n'est **pas** un endroit pour me questionner pour de l'aide. Je garde osu! support separé de mes messages personnelles, et cela est déjà un travail assez long. Je vous demande donc de suivre une de ces options. 
```

Comment:
Add the title to the top of this page (above this line) The English title is `# Do you really want to as peppy?`